Bugfix for ERROR Attempted method: security_groups_provider_updated and

VTabolin opened the pull request:
VimFaultException: The resource ID is in use

VTabolin committed:
Bugfix for ERROR Attempted method: security_groups_provider_updated and
VimFaultException: The resource ID is in use

VTabolin committed:
Improvements for increase stability

ekosareva commented on vmware_dvs/tests/unit/utils/test_dvs_util.py:
```diff
@@ -750,7 +750,8 @@ def test_creates_network_map_from_conf(self, *args):
             vmware_conf.api_retry_count,
-            vmware_conf.task_poll_interval)
+            vmware_conf.task_poll_interval,
+            pool_size=100)
```
Let's move pool_size constant it in config file(/vmware_dvs/common/config.py,  vmware_opts) and use smth like:
pool_size = vmware_conf.connections_pool_size

VTabolin replied:
good idea but I had no enough time on 200-nodes lab to do so

ekosareva commented on vmware_dvs/utils/dvs_util.py:
```diff
@@ -534,12 +552,15 @@ def blocked(self, value):
         config.api_retry_count,
-        config.task_poll_interval)
+        config.task_poll_interval,
+        pool_size=100)
```
Let's move pool_size constant it in config file(/vmware_dvs/common/config.py,  vmware_opts)

VTabolin replied:
done at next commit

VTabolin committed:
Added config parameter connections_pool_size

Removed hardcode parameter for vmware pool_size
Now it is a parameter in config file
